source/tit/sph/fluid_equations.hpp

```diff
@@ -134,7 +134,7 @@ class FluidEquations {
         // drho/dn = rho_0/(cs_0^2)*dot(g,n).
 #if EASY_DAM_BREAKING
         constexpr auto rho_0 = 1000.0;
-        constexpr auto cs_0 = 20 * sqrt(9.81 * 0.6);
+        const auto cs_0 = 20 * sqrt(9.81 * 0.6);
```

Comment:
Looks like the best options for us is to implement simple `constexpr` versions for some math functions.